README.md

```diff
@@ -62,28 +62,70 @@ tt convert -i <input .scc file> -o <output .ttml file>
 
 ### Command line
 
-```
-tt convert [-h] -i INPUT -o OUTPUT [--itype ITYPE] [--otype OTYPE] [--config CONFIG] [--config_file CONFIG_FILE]
-```
+`tt convert [-h] -i INPUT -o OUTPUT [--itype ITYPE] [--otype OTYPE] [--config CONFIG] [--config_file CONFIG_FILE]`
 
 * `--itype`: `TTML` | `SCC` | `STL` | `SRT` (extrapolated from the filename, if omitted)
 * `--otype`: `TTML` | `SRT` | `VTT` (extrapolated from the filename, if omitted)
 * `--config` and `--config_file`: JSON dictionaries with the following members:
-
   * `"general"."progress_bar": "true" | "false"`: whether a progress bar is displayed
```

Comment:
You may want to put the general configuration under a separate heading like you did for `imsc_writer` and  `stl_reader`.